docs/quickstart.rst

```diff
@@ -40,7 +40,7 @@ It looks something like this:
 Let's name this file ``example_bot.py``. Make sure not to name it ``discord.py`` as that'll conflict
 with the library.
 
-There's a lot going on here, so let's walk you through it step by step.
+There's a lot going on here, so let's walk you through it step by step:
```

Comment:
This didn’t need to be changed imo

Reply:
I thought it makes more sense, but I'm fine with both.